<ui version="4.0" >
 <class>layerDialog</class>
 <widget class="QDockWidget" name="layerDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>178</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Layer Dialog</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents" >
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <widget class="QTableWidget" name="layerTableWidget" />
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>28</width>
          <height>23</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="menuButton" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addButton" >
        <property name="text" >
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton" >
        <property name="text" >
         <string>-</string>
        </property>
        <property name="flat" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="logPlainTextEdit" />
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
